benchmark_runner/common/ocp_resources/create_ocp_resource_operations.py

```diff
@@ -53,33 +56,52 @@ def wait_for_ocp_resource_create(self, resource: str, verify_cmd: str, status: s
         """
         current_wait_time = 0
         while timeout <= 0 or current_wait_time <= timeout:
+            cmd = self.__oc.run(verify_cmd)
             # Count openshift-storage/ pv
             if count_openshift_storage:
-                if int(self.__oc.run(verify_cmd)) == self.__oc.get_num_active_nodes() * int(environment_variables.environment_variables_dict['num_odf_disk']):
+                if int(cmd) == self.__oc.get_num_active_nodes() * int(environment_variables.environment_variables_dict['num_odf_disk']):
                     return True
-                # Count disk maker (worker/master number * disk maker)
+            # Count disk maker (worker/master number * disk maker)
             elif count_disk_maker:
-                if int(self.__oc.run(verify_cmd)) == int(self.__oc.get_num_active_nodes()) * 2:
+                if int(cmd) == int(self.__oc.get_num_active_nodes()) * 2:
                     return True
-                # Count worker machines
+            # Count worker machines
             elif kata_worker_machine_count:
-                if int(self.__oc.run(verify_cmd)) > 0:
+                if int(cmd) > 0:
                     return True
-            # skip for count_openshift_storage
-            if not count_openshift_storage:
-                # verify query return positive result
-                if status:
-                    # catch equal or contains
-                    if status in self.__oc.run(verify_cmd):
-                        return True
-                else:
-                    if self.__oc.run(verify_cmd) != '':
+            # Count CSV
+            elif count_csv_names:
+                # Wait to CSV
+                if cmd:
+                    # ODF operator: not all CSV are started on the same time
+                    if resource == 'odf':
+                        # Wait for all CSVS
+                        if len(cmd.split()) == self.ODF_CSV_NUM:
+                            return True
+                    # default 1 CSV
+                    else:
                         return True
+            elif status:
+                # catch equal or contains
+                if status in cmd:
+                    return True
             # sleep for x seconds
             time.sleep(OC.SLEEP_TIME)
             current_wait_time += OC.SLEEP_TIME
         raise OCPResourceCreationTimeout(resource)
 
+    def verify_csv_installation(self, namespace: str, resource: str):
+        """
+        This method verifies csv installation
+        :param namespace:
+        :param resource:
+        """
+        csv_names = self.__oc.run(f"oc get csv -n {namespace} -ojsonpath={{$.items[*].metadata.name}}")
```

Comment:
`'-ojsonpath={{$.items[*].metadata.name}}'`

so no problems with metacharacter expansion.

Reply:
No already test it and it works